app/serializers/post_serializer.rb

```diff
@@ -331,7 +331,7 @@ def actions_summary
         summary[:count] = summary[:acted] ? 1 : 0
       end
 
-      summary.delete(:count) if summary[:count] == 0
+      summary.delete(:count) if summary[:count] == 0 || summary[:count].nil?
```

Comment:
```suggestion
      summary.delete(:count) if summary[:count].to_i.zero?
```

nil.to_i is 0 so this should work for both cases